src/renderers/dom/shared/CSSPropertyOperations.js

```diff
@@ -185,7 +185,7 @@ var CSSPropertyOperations = {
         styles[styleName],
         component
       );
-      if (styleName === 'float') {
+      if (styleName === 'float' || styleName === 'cssFloat') {
         styleName = styleFloatAccessor;
       }
```

Comment:
We only need to set styleName to cssFloat if it isn't already cssFloat, so this doesn't actually do anything. i appreciate the thought though.